_layouts/menu.html

```diff
@@ -16,15 +16,15 @@
       <li class="submenu-item">
         <a
           class="menu-link"
-          href="https://github.com/tc39/agendas/blob/HEAD/2022/09.md"
+          href="https://github.com/tc39/agendas/blob/main/2023/03.md"
         >
           {{ site["navigation"]["TC39"]["agenda"] }}
         </a>
       </li>
       <li class="submenu-item">
         <a
           class="menu-link"
-          href="https://github.com/tc39/notes/blob/HEAD/meetings/2022-06"
+          href="https://github.com/tc39/notes/tree/main/meetings/2023-01"
```

Comment:
```suggestion
          href="https://github.com/tc39/notes/tree/HEAD/meetings/2023-01"
```
why is this one "tree" and the other one "blob" above?

Reply:
`blob` is a file

Reply:
ah, so the previous link here was wrong?

Reply:
correct